src/Octoshift/GithubClient.cs

```diff
@@ -17,15 +17,15 @@ public class GithubClient
         private readonly HttpClient _httpClient;
         private readonly OctoLogger _log;
 
-        public GithubClient(OctoLogger log, HttpClient httpClient, string personalAccessToken)
+        public GithubClient(OctoLogger log, HttpClient httpClient, VersionChecker versionChecker, string personalAccessToken)
```

Comment:
same, unnecessary dependency.

Reply:
changed to IVersionProvider being injected